spec/acceptance/self_service_spec.rb

```diff
@@ -112,6 +112,19 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet resource service pe-postgresql ensure=running')
       end
+      it 'if S0012 conditions for false are met' do
+        run_shell('service puppet stop; puppet config set runinterval 20')
```

Comment:
What's the reasoning behind stopping the service here? Stopping/starting it would kick off an agent run and could interfere with other tests. Maybe `puppet agent --disable` and `puppet agent --enable` to prevent it from running instead of the service stop/start could satisfy the requirements?